src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -13579,6 +13584,10 @@ void Unit::ExitVehicle(Position const* /*exitPosition*/)
         return;
 
     GetVehicleBase()->RemoveAurasByType(SPELL_AURA_CONTROL_VEHICLE, GetGUID());
+    if (Player* player = ToPlayer())
+    {
```

Comment:
same here and everywhere else

Reply:
Done https://github.com/The-Cataclysm-Preservation-Project/TrinityCore/pull/369/commits/8442a06a243482d8ea714eef1b1fbb302f812fc0